packages/e2e/cypress/integration/dashboard.spec.ts

```diff
@@ -0,0 +1,43 @@
+import { SEED_PROJECT } from 'common';
+
+describe('Explore', () => {
+    before(() => {
+        // @ts-ignore
+        cy.login();
+        // @ts-ignore
+        cy.preCompileProject();
+
+        ['chat.lightdash.com', 'www.loom.com'].forEach((url) => {
+            cy.intercept(
+                {
+                    hostname: url,
+                },
+                (req) => {
+                    req.destroy();
+                },
+            );
+        });
```

Comment:
This is great! 😃 
We could have this for all the tests if we define it in the `cypress/support/index.js` ( [blog post explaining](https://ashley-sinner.medium.com/cypress-index-support-file-write-once-run-everywhere-4106c297a1aa) )